src/Hazelcast.Net/DistributedObjects/Impl/HTxMap.cs

```diff
@@ -91,28 +91,29 @@ public async Task<IReadOnlyList<TKey>> GetKeysAsync(IPredicate predicate)
         }
 
         public Task SetAsync(TKey key, TValue value)
-            => SetAsync(key, value, TimeToLive.InfiniteTimeSpan);
-
-        public Task<TValue> PutAsync(TKey key, TValue value)
-            => PutAsync(key, value, TimeToLive.InfiniteTimeSpan);
+            => SetAsync(key, value, TimeToLive.Infinite);
 
         public async Task SetAsync(TKey key, TValue value, TimeSpan timeToLive)
         {
             var (keyData, valueData) = ToSafeData(key, value);
-            var timeToLiveMilliseconds = timeToLive.CodecMilliseconds(-1);
+            // FIXME MEH TTL?
+            var timeToLiveMs = timeToLive.RoundedMilliseconds(false);
```

Comment:
I think we don't support TTL for TxnMap, so the codec does not have this parameter. I think we can remove it from this client too

Reply:
great, updating